service/metric/handler.go

```diff
@@ -0,0 +1,27 @@
+package metric
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Server is an http Metrics server.
+type Server struct {
+	metrics http.Handler
+}
+
+// NewServer returns a new metrics server.
+func NewServer() *Server {
+	return &Server{
+		metrics: promhttp.Handler(),
+	}
+}
+
+// ServeHTTP returns a metrics server.
+// Use it to set up the prometheus endpoint in your web service
+// router := mux.NewRouter()
+// router.Handle("/_status/metrics", metric.NewServer()).Methods("GET")
```

Comment:
I created an extra package only for the `http.Handler` so we can easily integrate the monitoring endpoint not depending on which HTTP router or framework is used.